### PR TITLE
Pre fill ticket services

### PR DESCRIPTION
**Pull request by jjessieshang**
Pre-filling tickets with cost estimates of services.

### CODE REVIEW

File: frontend/src/components/CostEstimate/index.js [hunk 1]
```diff
@@ -49,6 +53,7 @@ export const CostEstimateFull = () => {
       </span>
 
       <div className="CostEstimates">
+        { console.log(costEstimateBillablesMap)}
```
Comment (Harin329) [inline frontend/src/components/CostEstimate/index.js:56]: Can we remove this logging here :)

File: frontend/src/redux/reducers/costEstimateReducer.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { combineReducers } from "redux";
 import { TOGGLE_COST_ESTIMATE } from "../actions/uiActions";
-import { SET_COST } from "../actions/costActions";
+import { SET_COST} from "../actions/costActions";
```
Comment (Harin329) [inline frontend/src/redux/reducers/costEstimateReducer.js:3]: lint nit: space here

File: frontend/src/redux/reducers/ticketReducer.js [hunk 3]
```diff
@@ -121,7 +121,7 @@ const currentTicket = (state = null, action) => {
 };
 
 const currentTicketServiceCosts = (state = [], action) => {
-  switch (action.type) {
+  switch (action.type) {
```
Comment (Harin329) [inline frontend/src/redux/reducers/ticketReducer.js:124]: lint nit: space

File: frontend/src/redux/saga/surveySaga.js [hunk 4]
```diff
@@ -17,8 +18,28 @@ export function* submitResponseSaga({ payload }) {
...
+        billed: false,
+        billedTime: null,
+        createdBy: "USER-A"
+      } });
+    })
+  );
+
   yield all(
-    payload.formResponses.map((response) => {
+    payload.formResponses.map((response) => {
```
Comment (Harin329) [inline frontend/src/redux/saga/surveySaga.js:42]: space

File: frontend/src/screens/request-form/request-form.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ import { useDispatch, useSelector } from "react-redux";
 import "./request-form.css";
 import { appColor } from "../../constants";
 import { LOAD_QUESTION } from "../../redux/actions/questionActions";
-import { LOAD_COST } from "../../redux/actions/costActions";
+import { LOAD_COST} from "../../redux/actions/costActions";
```
Comment (Harin329) [inline frontend/src/screens/request-form/request-form.js:6]: space

File: frontend/src/redux/saga/surveySaga.js [hunk 6]
```diff
@@ -17,8 +18,28 @@ export function* submitResponseSaga({ payload }) {
     task_description: "Harin's Service Request (Replace this with description)",
     task_state: "open",
   });
+
+
+  yield all(
```
Comment (Harin329) [inline frontend/src/redux/saga/surveySaga.js:23]: Nice! Yield all sagas can get confusing, this looks good

**Review by Harin329 — CHANGES_REQUESTED** [review]
Looks good! Just some small linting changes, to try to keep the code base cleaner moving forward

**Comment by jjessieshang** [comment]
sounds good, will be more careful with the styling in the future :)

**Review by Harin329 — APPROVED** [review]

**Pull request merged**